app/jobs/labor/rate_limit_checker_job.rb

```diff
@@ -0,0 +1,10 @@
+module Labor
```

Comment:
I'm not sure if using `Labor` as a namespace is a good decision. There's is an `app/labor` folder, but it doesn't have any strong meaning.  Probably, I would go without a namespace for this one :thinking: